packages/server/lib/controllers/onboarding.controller.ts

```diff
@@ -161,12 +162,31 @@ class OnboardingController {
                 payload.progress = 3;
             }
 
-            const getRecords = await syncDataService.getAllDataRecords(connectionId, DEMO_GITHUB_CONFIG_KEY, environment.id, DEMO_MODEL);
-            if (!getRecords.success) {
+            let getRecords: Result<CustomerFacingDataRecord[], Error> = resultErr('failed_to_get_records');
```

Comment:
It's hiding the actual error (if any) but maybe it's alright during the transition?

Reply:
we don't do anything with the actual error but yeah fixed in 9e577400